devel/fortio/periodic.go

```diff
@@ -170,7 +170,7 @@ func (r *periodicRunner) Run() {
 	fmt.Printf("Ended after %v : %d calls. qps=%.5g\n", elapsed, functionDuration.Count, actualQPS)
 	if useQPS {
 		percentNegative := 100. * float64(sleepTime.hdata[0]) / float64(sleepTime.Count)
-		if percentNegative > 3 {
+		if percentNegative > 5 {
```

Comment:
I'll add a comment, thx, I think it's somewhat arbitrary  - it's how often we are falling behind for the target qps - 5% is in the noise (actually maybe 10) - the only effect is more verbose debugging and the warning